### DISCUSSION TITLE
Smooth scroll for macOS

### DISCUSSION THREAD

**Discussion by austincondiff** [migueldeicaza/SwiftTerm#300] [General]
Users/developers may enjoy configuring their terminal to use smooth scroll on macOS apps. Thoughts?

**Comment by migueldeicaza** [comment]
smooth scroll sadly has become a bit of an overloaded term.   And I used it in a way that is confusing to others.

When I implemented it for iOS I mean to say “UIScrollView-based content” which happens to give you up/down control via the system scrolling system.

vscode and the Dec vt100 have other slightly different interpretations, so I am not sure what you mean :-)

the Dec version is “it smoothly animates the new line operation”
